lib/icingadb/icingadb-utility.cpp

```diff
@@ -245,6 +245,17 @@ String IcingaDB::GetLowerCaseTypeNameDB(const ConfigObject::Ptr& obj)
 }
 
 long long IcingaDB::TimestampToMilliseconds(double timestamp) {
+	// In addition to the limits of the Icinga DB MySQL (0 - 2^64) and PostgreSQL (0 - 2^63) schemata,
+	// years not fitting in YYYY may cause problems, see e.g. https://github.com/golang/go/issues/4556.
+	// RFC 3339: "All dates and times are assumed to be (...) somewhere between 0000AD and 9999AD."
+	//
+	// The below limits include "better safe than sorry" buffers
+	// not to worry about time zones, floating point arithmetic details and the precise numbers themselves.
+	// The buffers were chosen so that the resulting numbers are easy to write and read.
```

Comment:
> The buffers were chosen so that the resulting numbers are easy to write and read.

This last comment doesn't make sense to me, so I would remove it.